drivers/local.go

```diff
@@ -35,12 +35,13 @@ func NewMemoryDriver(baseURI string) *MemoryOS {
 func (ostore *MemoryOS) NewSession(path string) OSSession {
 	ostore.lock.Lock()
 	defer ostore.lock.Unlock()
+	path = path + "/"
```

Comment:
mediaserver in `getHLSSegmentHandler` calls `GetSession` without end slash

Reply:
Fixed this in a better way: https://github.com/livepeer/go-livepeer/pull/575/commits/38892e07840ca40bf4a8df6dc365717d03132efa

I'm still not completely confident in this area of the code; we really need some unit tests here.